docs/Buttons.md

```diff
@@ -307,6 +307,16 @@ const MyEdit = () => (
 
 ### `<CloneButton>`
 
+## Performance
```

Comment:
Wasn't sure whether to have this in the button or datagrid component docs, or to create a new top-level page specifically for performance. Happy to go with whatever you folks feel is most appropriate